ooiservices/app/redmine/routes.py

```diff
@@ -187,14 +184,15 @@ def get_redmine_users():
                         status=400,
                         mimetype="application/json")
 
-    proj = request.args['project']
-    project = redmine.project.get(proj).refresh()
-
+    all_users = redmine.user.all(offset=1, limit=100)
```

Comment:
Why are we limiting the number of suers to 100?  This project will most likely have more than 100 users.... Am i misreading how this limit works?